docs/faq.md

```diff
@@ -109,6 +109,46 @@ commands =
 `tox` will not do any install. Poetry installs all the dependencies and the current package an editable mode.
 Thus, tests are running against the local files and not the built and installed package.
 
+#### Usecase #4
```

Comment:
I would prefer to call it something like "Complex example".

Reply:
I disagree pretty strongly, but I understand where you're coming from, in that it is more complex than the others.

`tox` can't use dependency groups, and I suspect we'd agree that dependency groups in poetry provide a better developer experience than extras. (Not to mention the issue that a published package exposes its extras.) Users of `tox` are _trying_ to do multi-tool and multi-version testing. My experience has been that their use-cases are generally much closer to this than the other ones. I could argue for calling it "Realistic example" (but I think that's a bad name and not productive for our conversation 😁 ).

Having laid out that reasoning, would you still like it renamed? I'm willing to do it, but wanted to air the counterpoint first.

Reply:
Having tinkered with this again and read the suggestion below of an added `pyproject.toml`, my perspective is moving away from my previous comment a little. I still think this is the best and most complete usage of tox+poetry we can present, but I definitely can acknowledge that it is more complicated than the other examples.

I renamed to `Complex Usage with Dependency Groups` for now.